src/msw/window.cpp

```diff
@@ -3557,6 +3601,17 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
             }
         }
         break;
+
+        case WM_TOUCH:
+        {
+            if ( !GestureFuncs::IsOk() )
+                break;
+
+            HandleTouch(wParam, lParam);
+            // Let DefWindowProc handle the touch events too
```

Comment:
It would be nice to explain why do we need it to always have them?

Reply:
The value of `processed` doesn't appear to have any effect here.

Reply:
If there is no good reason to _not_ use the usual logic, I'd stick to it. I.e. just do

```cpp
processed = HandleTouch(wParam, lParam);
```